source/main/modules/window-manager/index.ts

```diff
@@ -251,6 +285,13 @@ export default class WindowManager {
           win.close()
         }
       }
+      if (process.platform === 'win32' || process.platform === 'linux') {
+        if (global.config.get('system.leaveAppRunning')) {
```

Comment:
Fails `yarn lint` : `  289:13  warning  Unexpected any value in conditional. An explicit comparison or type cast is required  @typescript-eslint/strict-boolean-expressions`